podman/domain/images_manager.py

```diff
@@ -31,6 +32,8 @@
 
 logger = logging.getLogger("podman.images")
 
+builtin_list = list
```

Comment:
@inknos builtin_list doesn't is not used consistently in this file. Should all references to list() not associated with `def list()` be `builtin_list()`

Reply:
thanks @jwhonce. could you add the comments related to this [here](https://github.com/containers/podman-py/pull/476) please? This is just adding 2 commits on top of the changes of the other PR